docs/proposals/operator-managed-infra.md

```diff
@@ -2,73 +2,56 @@
 Design doc: Not a feature yet
 
 Instead of managing all the infrastructure through terraform, the
-operator itself should be responsible for managing the infra it requires,
-such as buckets, K8s Sevice Account workload identity mappings and Container Image
-Registry.
+operator itself should be responsible for managing the Workload Identity
+bindings between the K8s Service Account and the Cloud Service Account/IAM role.
 
-This will make it easier for users to adopt Substratus into existing
-environments without having to learn IaC tooling.
+Doing this, will make it possible to use Substratus resources in multiple
+namespaces without having to use another tool to configure Cloud identity/role mappings.
+
+Out of scope:
+* Managing the bucket, image registry or other any other resources
 
 
 ## Why?
-* Simplified install: Allow install to use `gcloud` or `eksctl` to create initial K8s cluster + nodepools
+* Simplified install: Allow install to use `gcloud/aws` or `eksctl` to create initial K8s cluster + nodepools
 * Make install onto existing K8s cluster straight forward and remove
   complexity of using Terraform
-* Improved UX, all an end-user needs to do is install our manifests and ensure GCP service account
-  has enough permissions
-* Enterprises generally have very specific requirements
-  about how they create their K8s Clusters and nodepools.
-* Allow Substratus to work in a multi namespace environment.
-  Currently namespace is hardcoded in workload identity settings in the terraform
+* Improved UX, all an end-user needs to do is install our manifests and ensure GCP service account has enough permissions
```

Comment:
> ensure GCP service account has enough permissions

I think fleshing out what's meant by this statement will answer what we want here: https://github.com/substratusai/substratus/pull/170#discussion_r1287980291